### PR TITLE
AA deployments

### PR DESCRIPTION
**Pull request by VictorCheng98**

### CODE REVIEW

File: deploy/2_deploy_SimpleAccountFactory.ts [hunk 1]
```diff
@@ -1,23 +1,22 @@
 import { ethers } from "hardhat";
 import { DeployFunction } from "hardhat-deploy/types";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { entryPoint07Address } from "viem/account-abstraction";
```
Comment (dingstacked) [inline deploy/2_deploy_SimpleAccountFactory.ts:4]: this name looks very weird wondering why they made this

File: hardhat.config.ts [hunk 2]
```diff
@@ -35,6 +36,9 @@ function getChainConfig(chain: keyof typeof chainIds): NetworkUserConfig {
     case "base-sepolia":
       jsonRpcUrl = "https://" + chain + ".g.alchemy.com/v2/" + alchemyApiKey;
       break;
+    case "night-testnet":
+      jsonRpcUrl = "https://rpc-night-testnet-brk51d2fuk.t.conduit.xyz";
```
Comment (dingstacked) [inline hardhat.config.ts]: nit: env?

File: hardhat.config.ts [hunk 3]
```diff
@@ -119,6 +125,14 @@ const config: HardhatUserConfig = {
           browserURL: "https://sepolia.basescan.org",
         },
       },
+      {
+        network: "night-testnet",
+        chainId: chainIds["night-testnet"],
+        urls: {
+          apiURL: "https://explorer-night-testnet-brk51d2fuk.t.conduit.xyz/api",
```
Comment (dingstacked) [inline hardhat.config.ts]: same here

File: hardhat.config.ts (hunk 2 above)
Comment (VictorCheng98) [reply hardhat.config.ts]: i think the env should be ok its the public rpc endpoint

Comment (VictorCheng98) [reply hardhat.config.ts]: actually eh yea ill change it

File: deploy/2_deploy_SimpleAccountFactory.ts (hunk 1 above)
Comment (VictorCheng98) [reply deploy/2_deploy_SimpleAccountFactory.ts:4]: its an address constant that should be the same on all chains

**Review by dingstacked — APPROVED** [review]

**Pull request merged**